src/Kernel/RuntimeSyntaxError.class.st

```diff
@@ -0,0 +1,10 @@
+"
+When compiling syntactically incorrect code, we compile raising this exception.
+
+This way the debugger opens and the programmer can easily fix the faulty method
+"
+Class {
+	#name : #RuntimeSyntaxError,
+	#superclass : #Halt,
```

Comment:
I think it should be an Error subclass. Why you decide to make it a Halt?

Reply:
Hmm... right. My brain thinks Halt opens the debugger because I want to, while error is an error. And I wanted to open debugger :-)
But then, this is actually a error, and error opens the debugger, too.

So yes, should be Error subclass